src/test/java/com/epam/qalab/BaseTest.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.qalab;
+/* 
+@author
+Marat Lagun
+created on 3/2/22   
+*/
+
+import com.epam.qalab.factory.WebDriverFactory;
+import org.openqa.selenium.WebDriver;
+
+public class BaseTest {
+    protected final WebDriver driver = new WebDriverFactory().getWebDriver();
+    String incorrect = "qwe4t5et6789tz3js4jdm1234567890";
+    String correct = "laptop";
```

Comment:
You probably could move these class variables (incorrect, correct) to your test file (SearchFieldTest.java), because they are only related to this particular test. 
There is no need to force parent class to include them.